bundles/org.openhab.binding.pihole/src/main/java/org/openhab/binding/pihole/internal/rest/model/DnsStatistics.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.pihole.internal.rest.model;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * @author Martin Grzeslowski - Initial contribution
+ */
+@NonNullByDefault
+public record DnsStatistics(@SerializedName("domains_being_blocked") @Nullable Integer domainsBeingBlocked,
+        @SerializedName("dns_queries_today") @Nullable Integer dnsQueriesToday,
```

Comment:
Using `new GsonBuilder().setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES).build()` you could save many `@SerializedName`

Reply:
This is makred as resovled without a follow up (as it seems to me)

Reply:
@magx2 can you comment?

Reply:
you are looking on an outdated code